tests/filesTest.js

```diff
@@ -943,6 +945,8 @@ describe('Main: Currently testing files management,', function () {
     })
   })
 
+  // [OCIS] ocis returns `text/plain` content-type and not `text/html` like oc10 while sending move request
+  // https://github.com/owncloud/ocis/issues/1793
```

Comment:
as `MOVE` request doesn't send any payload (response body), it will also be safe to remove `headers: htmlResponseHeaders` from here https://github.com/owncloud/owncloud-sdk/blob/e3237cbbadf8a61369d0a219572eadeda907a535/tests/filesTest.js#L982-L985